compiler/daml-extension/yarn.lock

```diff
@@ -8,17 +8,17 @@
   integrity sha512-RSu6I5yVVtjDP6PTkhDVvqOC+KJf8t4Uo2qCLxGjunLVt4guqKLt3KQX+dEU/qVn5vECXGTprDSNk+c9G6hDmg==
 
 "@types/node-fetch@^2.5.3":
-  version "2.5.10"
-  resolved "https://registry.yarnpkg.com/@types/node-fetch/-/node-fetch-2.5.10.tgz#9b4d4a0425562f9fcea70b12cb3fcdd946ca8132"
-  integrity sha512-IpkX0AasN44hgEad0gEF/V6EgR5n69VEqPEgnmoM8GsIGro3PowbWs4tR6IhxUTyPLpOn+fiGG6nrQhcmoCuIQ==
+  version "2.5.12"
```

Comment:
I got tired of doing selective updates and just regenerated the stupid yarn.lock. Verified manually that it still works.

Reply:
except ofc this doesn’t compile on Windows so reverted